### DISCUSSION TITLE
https://shlink.io/HnuHa/login  >>> https://example.com/login

### DISCUSSION THREAD

**Discussion by dossif** [shlinkio/discussions#76] [Help wanted] [answered]
Hi,

I installed shlink v3.0.3.

And I created new short-url like https://shlink.io/HnuHa with long-url https://example.com

Now https://shlink.io/HnuHa redirected to https://example.com
But https://shlink.io/HnuHa/login does not redirect to https://example.com/login and https://shlink.io returned 404 :(

Is it expected behavior?

**Comment by acelaya** [comment, marked as answer]
Yes, this is the default behavior, but Shlink can be configured to behave like you describe https://shlink.io/documentation/some-features/#extra-path-forwarding

**Reply by dossif** [reply]
I added ENV REDIRECT_APPEND_EXTRA_PATH=true to docker-compose and it is works! Thanks!
